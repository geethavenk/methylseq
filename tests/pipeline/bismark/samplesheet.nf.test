nextflow_pipeline {

    name "Test Replicate Handling"
    script "main.nf"

    tag "pipeline"

    test("Should handle replicates and works") {
        when {
            params {
                aligner        = "bismark"
                save_reference = true
                input          = "$baseDir/tests/pipeline/bismark/assets/samplesheet.csv"
                outdir         = "$outputDir"
            }
        }

        then {
            assertAll(
                { assert workflow.success }
            )
        }
    }
}
