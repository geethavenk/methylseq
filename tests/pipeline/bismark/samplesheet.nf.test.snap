{
    "single_end_software_versions": {
        "content": [
            "{BISMARK_ALIGN={bismark=0.24.0}, BISMARK_GENOMEPREPARATION={bismark=0.24.0}, FASTQC={fastqc=0.12.1}, QUALIMAP_BAMQC={qualimap=2.2.2-dev}, TRIMGALORE={trimgalore=0.6.7, cutadapt=3.4}, Workflow={nf-core/methylseq=v2.7.0dev}}"
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.0"
        },
        "timestamp": "2024-05-17T14:37:44.368983"
    }
}